assets/src/components/MediaEmbedPreview/MediaEmbedPreview.js

```diff
@@ -0,0 +1 @@
+export const MediaEmbedPreview = ({ html }) => <div dangerouslySetInnerHTML={{ __html: html }}></div>;
```

Comment:
This component is so simple that it's probably easier to include its contents directly. It would also show more clearly that a `**dangerously**SetInnerHTML` is happening.

Reply:
Good point! Updated 🙂